src/packages/utils/package.json

```diff
@@ -57,6 +57,7 @@
   "scripts": {
     "build:umd": "cross-env NODE_ENV=production webpack --config webpack.prod.config.js",
     "build:umd:withAnalyzer": "cross-env NODE_ENV=production ANALYZER=true webpack --config webpack.prod.config.js",
+    "install:deps": "yarn --cwd ../ install --frozen-lockfile && yarn install --frozen-lockfile && yarn --cwd ../../../ install --frozen-lockfile",
```

Comment:
This shouldn't be needed since its an independent package, any specific reason for adding this ?

Reply:
You are correct.  Removed in eb76191.